service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -72,6 +72,7 @@
   private final AuthenticatedUserRequestFactory authenticatedUserRequestFactory;
   private final HttpServletRequest request;
   private final ReferencedResourceService referenceResourceService;
+  private final Logger logger = LoggerFactory.getLogger(WorkspaceApiController.class);
```

Comment:
Since this is an `@Component` singleton, does it make any difference to make it static?

Reply:
I don't believe so. I don't see a pattern in our usage today. It's about a 60/40 split. My philosophy is that anything that could be static might as well be, since it makes refactoring (like moving constants and methods) easier.

Reply:
Sounds reasonable. `static` it is!